example.docker.env

```diff
@@ -0,0 +1,7 @@
+GOTRUE_SITE_URL="http://localhost:3000"
```

Comment:
Should this contain more of the values from `example.env`? I'm thinking someone's setup process might be as follows:

1. Clone the repo
2. Decide to run locally or in docker
3. Copy the appropriate env example to `.env`
4. Run GoTrue

This looks like only select vars will be overridden? Just curious on how this will work out. I'm guessing we wont want to work directly from `example.docker.env` because it would get checked into the repo along with someone's credentials within it.

Reply:
yeah the `example.docker.env` should basically be a copy of `example.env` and anyone starting with the docker setup should create a `.env.docker` which would be git ignored - will add this to the readme and contributor's guide!